canonical-web-and-design/maas-ui
# Allow middleware checks to be disabled

## Done
- Add and env option to disabled the redux toolkit middleware checks.

## QA
- View the machine list in development, you should see a flood of middleware warnings.
- Create a file called ui/.env.development.local and insert `REACT_APP_CHECK_MIDDLEWARE=false`.
- Run the app again (you'll need to stop and start your dotrun or yarn).
- The machine list should not have the warnings.

## Fixes
Fixes: https://github.com/canonical-web-and-design/maas-ui/issues/1045.
